specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,205 @@
+# Semantic conventions for browser events
```

Comment:
This should make use of semantic conventions YAML definitions for attributes, where possible.